code/modules/mob/living/carbon/carbon_movement.dm

```diff
@@ -17,6 +17,9 @@
 		if(legcuffed)
 			. += legcuffed.slowdown
 
+	if(stat == SOFT_CRIT)
+		. = max(SOFTCRIT_MIN_SLOWDOWN, . + SOFTCRIT_ADD_SLOWDOWN) //regardless of how fast you are, you move at a maximum of SOFTCRIT_MIN_SLOWDOWN while in softcrit
```

Comment:
Hmm; this is supposed to run after everything else modifies speed, so that the max() properly limits speed. How inconvenient.

Reply:
or you could just change the code :thinking: